commit 8dcf62cdeb5ae73e5d60688295a868756a6416f4
Author: Marco Martin <notmart@gmail.com>
Date:   Thu Sep 5 11:45:18 2013 +0200

    reload menu entries on config changed
    
    if the config has been changed by a script, reload the menu entries, in case the enabled ones were changed

diff --git a/plasma/netbook/containments/sal/sal.cpp b/plasma/netbook/containments/sal/sal.cpp
index 266deb9..2102e04 100644
--- a/plasma/netbook/containments/sal/sal.cpp
+++ b/plasma/netbook/containments/sal/sal.cpp
@@ -60,6 +60,7 @@
 SearchLaunch::SearchLaunch(QObject *parent, const QVariantList &args)
     : Containment(parent, args),
       m_runnerModel(0),
+      m_serviceModel(0),
       m_backButton(0),
       m_queryCounter(0),
       m_maxColumnWidth(0),
@@ -275,6 +276,9 @@ void SearchLaunch::configChanged()
         }
     }
 
+    if (m_serviceModel) {
+        m_serviceModel->setPath("/");
+    }
     m_stripUninitialized = false;
 }
 
